src/events_block_comment_move.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * @license
+ * Copyright 2024 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as Blockly from "blockly/core";
+import { BlockCommentBase } from "./events_block_comment_base.js";
+
+class BlockCommentMove extends BlockCommentBase {
+  constructor(opt_blockComment, oldCoordinate, newCoordinate) {
+    super(opt_blockComment);
+    this.type = "block_comment_move";
+    this.oldCoordinate_ = oldCoordinate;
+    this.newCoordinate_ = newCoordinate;
+  }
+
+  toJson() {
+    return {
+      ...super.toJson(),
+      newCoordinate: {
+        x: this.newCoordinate_.x,
+        y: this.newCoordinate_.y,
+      },
```

Comment:
Can this just be `newCoordinate: this.newCoordinate_`?

Reply:
Dibe